terraform/modules/kubernetes/main.tf

```diff
@@ -187,13 +187,26 @@ resource "kubernetes_deployment" "main_worker" {
       spec {
         node_selector = {
           "kubernetes.azure.com/agentpool" = "applications"
-          "kubernetes.io/os" = "linux"
+          "kubernetes.io/os"               = "linux"
         }
         container {
           name    = local.worker_name
           image   = var.app_docker_image
           command = ["bundle"]
           args    = ["exec", "sidekiq", "-c", "5", "-C", "config/sidekiq-main.yml"]
+          liveness_probe {
```

Comment:
Can we add a failure threshold as we did for the webapp? for both liveness_probe and startup_probe
This would reduce false positives